### PR TITLE
Add meterial 2

### PR DESCRIPTION
**Pull request by DavidBronfen**
#5

### CODE REVIEW

**Comment by DavidBronfen** [comment]
Trying to set the material theme.
Get this error after compiling 
image

File: client/src/app/app.module.ts [hunk 1]
```diff
@@ -12,7 +15,8 @@ import { AppComponent } from './app.component';
   imports: [
     BrowserModule,
     FormsModule,
-    HttpModule
+    HttpModule,
+    MaterialModule
```
Comment (nirgn975) [inline client/src/app/app.module.ts]: Add a comma at the end

File: client/src/styles.scss [hunk 2]
```diff
@@ -1 +1,24 @@
...
+// Plus imports for other components in your app.
+
+// Include the base styles for Angular Material core. We include this here so that you only
+// have to load a single css file for Angular Material in your app.
+@include mat-core();
+
+// Define the palettes for your theme using the Material Design palettes available in palette.scss
+// (imported above). For each palette, you can optionally specify a default, lighter, and darker
+// hue.
+$retrogusto-app-primary: mat-palette($mat-indigo);
```
Comment (nirgn975) [inline client/src/styles.scss]: This should be your primary color of the app (the primary color your components will use).
If I understand things currently, you are not planning to use this `indigo` color, and use `#B2C1BE` instead. If this is the case, try to 
```
$retrogusto-app-primary: mat-palette(mat-color('#B2C1BE'));
```

**Comment by DavidBronfen** [comment]
@nirgn975 I think I didn't get the point of these theme thing, what does it serve for?
And if i have more than 3 colors that want to use in my app, where and how do i set them?

**Comment by DavidBronfen** [comment]
@nirgn975 
Any idea why is it fails?
I know it's because of the color i have changed..

**Comment by nirgn975** [comment]
@DavidBronfen You need to use a variable of color and not an hex code.
There isn't any option (yet) to give it a custom color, so what we'll do is give it some genraic color. let's say:
```
$retrogusto-app-primary: mat-palette($mat-blue);
$retrogusto-app-accent:  mat-palette($mat-amber);
```

And then add a custom css classes that you'll use across your app:
```
.primary-color {
    color: #3F51B5 !important;
  }
.accent-color {
    color: #FF4081 !important;
  }
.background-primary-color {
    background-color: #3F51B5 !important;
  }
.background-accent-color {
    background-color: #FF4081 !important;
  }
.white-text {
    color: #FFF !important;
}
```
^ This is just an example.

When you'll create components it will look something like this:
```
<button md-button class="background-primary-color white-text">I'm a basic button with</button>
```

**Comment by DavidBronfen** [comment]
@nirgn975 
> And then add a custom css classes that you'll use across your app:

This content will be in the `vars.scss` file? or new file called lets say `theme-colors.scss`?

**Comment by nirgn975** [comment]
@DavidBronfen They should by in the `client/src/styles.scss` after all you material settings.

**Pull request merged**